packages/eslint-config-udemy-jasmine-addons/README.md

```diff
@@ -1,4 +1,26 @@
-# ESLint Config Udemy Jasmine
+Udemy ESLint Config for Jasmine
+===============================
 
-This config groups [*Jasmine*](https://github.com/jasmine/jasmine) specific rules, 
-and used by [`eslint-config-udemy`](packages/eslint-config-udemy).
+This configuration groups [*Jasmine*](https://github.com/jasmine/jasmine) specific ESLint rules.
+It is recommended to be used by any Jasmine codebase developed for Udemy. 
```

Comment:
I'd delete this whole line.

Reply:
Same comment for other files.

Reply:
I want to emphasize the fact that these addons are supposed to be re-used, and are not only meant for extended by basics.